src/main/sensors/barometer.h

```diff
@@ -55,7 +55,7 @@ PG_DECLARE(barometerConfig_t, barometerConfig);
 
 typedef struct baro_s {
     baroDev_t dev;
-    int32_t BaroAlt;
+    int32_t BaroAlt;                     //altitude in centimeters (0.01m)
```

Comment:
Space between // and comment, and maybe use a capital A in "altitude" for consistency with the comments below.